pkgs/development/libraries/libtickit/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, libtermkey
+, unibilium
+, libtool
+}:
+stdenv.mkDerivation rec {
+  pname = "libtickit";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "leonerd";
+    repo = "libtickit";
+    rev = "v${version}";
+    hash = "sha256-QCrym8g5J1qwsFpU/PB8zZIWdM3YzOySknISSbQE4Sc=";
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+    "LIBTOOL=${lib.getExe libtool}"
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libtermkey unibilium libtool ];
```

Comment:
Are you sure `libtool` is a runtime dependency? It typically is only used at build-time, in which case it would go to `nativeBuildInputs`.